### ISSUE TITLE
Question to access urls

### ISSUE THREAD

**Issue by saschastahl** [rmatev/ganga-tools#1]
When I do  `download.get_access_urls(utils.outputfiles(jobs(0),"DV_Prompt_D02KsPiPi_DT.root"))`
is the order of subjobs respected in the output? I am wondering because I see some failures when looping over this list later and would like to be able to check the corresponding job.

**Comment by rmatev** [comment]
Both `download.get_access_urls` and `utils.outputfiles` should preserve order.

**Comment by saschastahl** [comment]
And if a subjob did not produce output this is skipped?

**Comment by rmatev** [comment]
Exactly, `utils.outputfiles` will skip it and issue an info/warning message

**Comment by saschastahl** [comment]
so my list later then loses the information which files were skipped?
